core/lib/snapshots_applier/src/tests/mod.rs

```diff
@@ -1,5 +1,7 @@
 //! Snapshot applier tests.
 
+// FIXME: test dropping preimages for version 0 snapshot
```

Comment:
Will this be addressed in this PR?

Reply:
It is already addressed (`applier_recovers_v0_snapshot` test with `drop_storage_key_preimages` set), I just forgot to remove the FIXME.